nipype/algorithms/confounds.py

```diff
@@ -985,7 +988,7 @@ class NonSteadyStateDetector(BaseInterface):
     def _run_interface(self, runtime):
         in_nii = nb.load(self.inputs.in_file)
         global_signal = (
-            in_nii.get_data()[:, :, :, :50].mean(axis=0).mean(axis=0).mean(axis=0)
+            in_nii.dataobj[:, :, :, :50].mean(axis=0).mean(axis=0).mean(axis=0)
```

Comment:
just to clarify `dataobj` is unscaled, right? if so, shouldn't this be `get_fdata()`

Reply:
`dataobj` is scaled, but the type doesn't change if the scale factors are 0/1.

This saves memory while retrieving values exactly identical to calling `get_data()[...]`.